README.md

```diff
@@ -34,7 +34,7 @@ It creates a Deployment which runs the application Pod, and then exposes that po
 | image | The fully qualified name of the application image | **Must be provided** |
 | image_pull_secret_name | In the case of private images, provide the image pull secret name if you have already created that. Otherwise, see [using-private-images](#using-private-images) | None
 | namespace | OpenShift project/Kubernetes namespace to target | Current context |
-| port | The port to expose from the application container | **Must be provided** |
+| port | A port to expose from the application container through a Service. If exactly one port is exposed in the container metadata, then this input can be omitted | None |
```

Comment:
can you put a period at the end of each input description?

Reply:
Okay :+1: